pallets/pallet-dyn-filter/src/lib.rs

```diff
@@ -0,0 +1,174 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2022 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+//! # DID lookup pallet
+//!
+//! This pallet stores a map from account IDs to DIDs.
+//!
+//! - [`Pallet`]
```

Comment:
Just leaving this comment here for later. I understand this is irrelevant for the review right now.

Reply:
good catch totally forgot to update that...

Reply:
[7d2ddc0](https://github.com/KILTprotocol/kilt-node/pull/397/commits/7d2ddc05ea3d21a15a0f766444934b2c8c37ac97)